plugin/src/main/kotlin/com/vanniktech/maven/publish/MavenPublishBaseExtension.kt

```diff
@@ -151,6 +151,10 @@ abstract class MavenPublishBaseExtension(
     // without afterEvaluate https://github.com/gradle/gradle/issues/12259 will happen
     project.afterEvaluate {
       pom { pom ->
+        val packaging = project.findOptionalProperty("POM_PACKAGING")
+        if (packaging != null) {
+          pom.packaging = packaging
```

Comment:
You mean this, right? No idea - maybe @gabrielittner  knows

Reply:
Removing this for now.